phpBB/includes/acp/acp_extensions.php

```diff
@@ -276,6 +276,12 @@ public function main_mode($id, $mode)
 				{
 					$this->template->assign_var('MIGRATOR_ERROR', $e->getLocalisedMessage($this->user));
 				}
+				catch (\Exception $e)
+				{
+					$stack_trace = phpbb_filter_root_path(str_replace("\n", '<br>', $e->getTraceAsString()));
+					$message = $e->getMessage();
+					$this->template->assign_var('MIGRATOR_ERROR', '<b>' . $message . '</b><br><br>' . $stack_trace);
```

Comment:
As mentioned above separate error from stack trace and move html to actual html file